packages/aws-cdk/lib/contextplugins.ts

```diff
@@ -48,12 +49,69 @@ export class SSMContextProviderPlugin implements ContextProviderPlugin {
     }
 }
 
+/**
+ * Plugin to read arbitrary SSM parameter names
+ */
+export class HostedZoneContextProviderPlugin implements ContextProviderPlugin {
```

Comment:
An instance of this needs to be added to `availableContextProviders` in `cdk.ts` as well.